### PR TITLE
Add support for validating the hd claim

### PR DESCRIPTION
**Pull request by neuroid**

### CODE REVIEW

**Comment by neuroid** [comment]
Hi @imkira. Any interest in merging this?

File: config.go [hunk 1]
```diff
@@ -19,6 +19,8 @@ var (
 	listenAddr     = flag.String("listen-addr", "0.0.0.0", "Listen address")
 	listenPort     = flag.String("listen-port", "", "Listen port (default: 80 for HTTP or 443 for HTTPS)")
 	audiences      = flag.String("audiences", "", "Comma-separated list of JWT Audiences (elements can be paths like \"/projects/PROJECT_NUMBER/apps/PROJECT_ID\" or regular expressions like \"/^\\/projects\\/PROJECT_NUMBER/.*\" if you enclose them in slashes)")
+	domains        = flag.String("domains", "", "Comma-separated list of allowed JWT hd claims (optional)")
```
Comment (imkira) [inline config.go]: nit: I would write `Hosted Domains (hd)` instead

**Review by imkira — CHANGES_REQUESTED** [review]

**Comment by imkira** [comment]
@neuroid thanks for the contribution. Looks good. Just have a minor comment regarding the flag description.
Cheers.

File: config.go (hunk 1 above)
Comment (neuroid) [reply config.go]: Sure

**Comment by neuroid** [comment]
@imkira 👋

**Review by imkira — APPROVED** [review]

**Comment by imkira** [comment]
Thanks for pinging me and apologies for the delay. Once again thanks for the contribution!

**Pull request merged**